portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyTestFile.testcase

```diff
@@ -0,0 +1,167 @@
+@component-name = "portal-web-forms-and-data-lists"
+definition {
+property testray.main.component.name = "Forms";
+    setUp {
+        TestCase.setUpPortalInstance();
+        User.firstLoginPG();
+        Navigator.openURL();
+
+}
+
+tearDown {
+
+    User.logoutAndLoginPG(
+        userLoginEmailAddress = "test@liferay.com",
+        userLoginFullName = "Test Test");
+}
+
+@description = "Access Portal Global Menu and validate CONFIGURATION string at ControlPanel"
+@priority = "5"
+test ValidateConfigurationStringAtControlPanel {
+        Click(locator1 = "//button[@data-qa-id='applicationsMenu']");
+
+        Click(locator1 = "//li[@id='_com_liferay_product_navigation_applications_menu_web_internal_portlet_ProductNavigationApplicationsMenuPortlet_tab_2']");
+
+        Pause(locator1 = "6000");
+
+        AssertElementPresent(locator1 = "//h2[@id='nav_control_panel.configuration']");
```

Comment:
Fix this testcase by splitting the poshi files into their given layers: testcase, macro and path.